samples/bluetooth/direct_test_mode/src/fem/fem.h

```diff
@@ -59,91 +66,64 @@ int fem_power_up(void);
  */
 int fem_power_down(void);
 
-/**@brief Set the front-end module to TX mode (PA).
- *
- * @note Transition from the Program State to the Transmit State takes
- *       a specific settle time for the device (in microseconds) and is triggered by
- *       the PA pin. Check your FEM specification to obtain that value.
- *       Suitable timing can be achieved by the @p active_delay parameters.
- *       This function triggers the @ref NRF_RADIO_TASK_TXEN task when activate_event
- *       occurs or triggers it immediately when @ref FEM_EXECUTE_NOW is used.
- *
- * @param[in] activate_event An event that triggers start of procedure - this
- *                           event will be connected to appropriate PPI channel.
- *                           @ref FEM_EXECUTE_NOW value starts the
- *                           procedure immediately.
- * @param[in] deactivate_event An event that triggers deactivation of the Tx -
- *                             this event will be connected to appropriate PPI
- *                             channel.
- * @param[in] activation_delay Tx PA activation delay between
- *                             @ref NRF_RADIO_TASK_TXEN and Tx pin setting.
+/**@brief Configure Tx gain of the front-end module in arbitrary units.
+ *
+ * @param[in] gain Tx gain in arbitrary units.
  *
  * @retval 0 If the operation was successful.
  *           Otherwise, a (negative) error code is returned.
  */
-int fem_tx_configure(uint32_t activate_event, uint32_t deactivate_event,
-		     uint32_t activation_delay);
+int fem_tx_gain_set(uint32_t gain);
```

Comment:
Isn't this function duplicated now? Line 76 and 117

Reply:
Well spotted. Thanks